src/services/jwt.service.js

```diff
@@ -0,0 +1,39 @@
+import jwt from 'jsonwebtoken';
+
+export const jwtService = {
+  sign(user) {
+    const token = jwt.sign(user, process.env.JWT_KEY, { expiresIn: '10s' });
```

Comment:
The access token expiration is set to '10s', which is quite short for most applications. Consider increasing the expiration time to a more practical duration, such as '15m' or '1h', depending on your application's needs.